src/diagnostics.ts

```diff
@@ -641,7 +636,10 @@ export class DiagnosticsManager {
             // Render diagnostics
             this.killAllButton.hide();
             verificationDiagnostics.renderIn(this.target);
-            if (verificationDiagnostics.hasErrors()) {
+            if (crashed) {
+                this.verificationStatus.text = `$(warning) Prusti crashed`;
```

Comment:
What about this message?
```suggestion
                this.verificationStatus.text = `$(error) Verification of ${target} '${escapedFileName}' failed with an unexpected error`;
```